Robus/inc/timestamp.h

```diff
@@ -21,6 +21,7 @@ typedef struct timestamp_token
     struct timestamp_token *next;
 } timestamp_token_t;
 
+#define TIMESTAMP_PROTOCOL 0xF
```

Comment:
This value is more related to messages information than the timestamp.
I think we should declare it close to the msg_t struct.

Also, we have 2 solutions to deal with this value : 
 - We create a list of values as you just did. In this case, we have to make an enum with DEFAULT_PROTOCOL in position 0 and timestamp next.
 - We convert `header.protocol` as a bitfield and reserve 3 bits for a protocol list, and 1 bit for the timestamp. If we choose this one we suppose that all the next revisions will have timestamped or non timestamped values.

I vote for the ENUM solution. We will change it for the bitfield solution in the future if we need it.